<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="RAKSHITHAs-MacBook-Air.local" failures="1" tests="4" name="Test" time="71.002" errors="0" timestamp="2022-04-13T15:06:09 IST">
  <testcase classname="testcases.LoginTest" name="ExternalData" time="6.588">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.relevantcodes.extentreports.ExtentTest.log(com.relevantcodes.extentreports.LogStatus, String, String)&amp;quot; because &amp;quot;this.test&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.relevantcodes.extentreports.ExtentTest.log(com.relevantcodes.extentreports.LogStatus, String, String)" because "this.test" is null
at pages.LoginPage.LoginFunction(LoginPage.java:50)
at testcases.LoginTest.ExternalData(LoginTest.java:77)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- ExternalData -->
  <testcase classname="testcases.LoginTest" name="LoginFailureTest" time="9.754"/>
  <testcase classname="testcases.LoginTest" name="LoginSuccessTest" time="8.193"/>
  <testcase classname="testcases.LoginTest" name="ParametereizedTest" time="8.977"/>
</testsuite> <!-- Test -->
